src/site/assignments/maze-solver-too/Assignment3/test/AStarMazeSolverTest.java

```diff
@@ -14,85 +14,50 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class AStarMazeSolverTest {
-    private static final String MAZE_NO_SOLUTION =
-            "#####\n" +
-                    "#   #\n" +
-                    "#   #\n" +
-                    "S  #E\n" +
-                    "#   #\n" +
-                    "#####";
+    private static final String MAZE_NO_SOLUTION = "#####\n" + "#   #\n" + "#   #\n" + "S  #E\n" + "#   #\n" + "#####";
```

Comment:
You may want to wrap these in:
```java
// @formatter:off
...code...
// @formatter:on
```
The veritcal spacing is intentional here for readability, the formatter can't know that.

You may need to enable the intellij setting for these comments to work: https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#exclude_part_of_code

Reply:
snazzy